test/run_test.py

```diff
@@ -90,6 +98,8 @@ def skip_test_p(name: str) -> bool:
             rc |= any(name.startswith(pattern) for pattern in blocklisted_patterns)
         if blocklisted_tests is not None:
             rc |= name in blocklisted_tests
+        if is_gfx94x_arch and os.environ.get('TEST_CONFIG') is not None and os.environ['TEST_CONFIG'] == 'inductor' and blocklisted_tests_inductor is not None:
```

Comment:
@BLOrange-AMD Should it have been `is_gfx94x_arch()` here?

Reply:
Resolved by PR: https://github.com/ROCm/pytorch/pull/1413